ios/RNSScreenStackAnimator.m

```diff
@@ -202,14 +208,30 @@ - (void)animateFadeFromBottomWithTransitionContext:(id<UIViewControllerContextTr
   CGAffineTransform topBottomTransform =
       CGAffineTransformMakeTranslation(0, 0.08 * transitionContext.containerView.bounds.size.height);
 
+  // proportions to default transition duration
+  NSDictionary *proportionToDefaultTransitionDuration = @{
+    @"slideOpen" : @1, // 0.35 s
+    @"fadeOpen" : @0.571, // 0.2 s
+    @"slideClose" : @0.714, // 0.25 s
+    @"fadeClose" : @0.428, // 0.15 s
+    @"fadeCloseDelay" : @0.286, // 0.1 s
```

Comment:
przyjdzie Kuba i się będzie zastanawiał

Reply:
Fixed in 23b136a1f04361931181c4225dc4b24463d079ba